src/Psalm/Internal/LanguageServer/Server/TextDocument.php

```diff
@@ -286,6 +299,14 @@ public function completion(TextDocumentIdentifier $textDocument, Position $posit
         return new Success(new CompletionList($completion_items, false));
     }
 
+    /**
+     * The signature help request is sent from the client to the server to request signature
+     * information at a given cursor position.
+     *
+     * @param TextDocumentIdentifier $textDocument
+     * @param Position $position
+     * @return Promise
```

Comment:
```suggestion
```
This doesn't add any additional information that was not provided by the function signature alread.

Reply:
Heh. You remind me of my coworker (not a bad thing!). He hates superfluous docblock types! I will remember that for the future :-)

Reply:
They tend to get out of sync, and Psalm doesn't flag `@param` entries for params that got removed. Speaking of which, I should probably file that as an issue, as they don't seem to be used in any way.